components/compliance-service/api/tests/02_nodes_spec.rb

```diff
@@ -658,6 +658,19 @@
     }.to_json
     assert_equal_json_sorted(expected_nodes, actual_nodes.to_json)
 
+    # Test filters by skipped status - expect 0 nodes, but expect counts to not be 0
+    actual_nodes = GRPC reporting, :list_nodes, Reporting::Query.new(filters: [
+      Reporting::ListFilter.new(type: 'environment', values: ['DevSec Prod beta']),
+      Reporting::ListFilter.new(type: 'status', values: ['skipped']),
+      Reporting::ListFilter.new(type: 'end_time', values: ['2018-03-04T23:59:59Z'])
+    ])
+    expected_nodes = {
+      "total" => 2,
+      "totalFailed" => 1,
+      "totalPassed" => 1
```

Comment:
Can we explicitly put in the empty array of nodes or will that cause a problem? Without it is is kinda hard to understand that you are also asserting that the nodes array is empty.  Perhaps moving the bit in the comment above after the `-` down here would help if we can't explicitly put an empty array of nodes in this assertion.

Reply:
the empty list is being returned; it's just the test that doesnt show it bc of how it translates the stuff to json from grpc

Reply:
<img width="682" alt="Screen Shot 2020-05-19 at 08 59 39" src="https://user-images.githubusercontent.com/10341541/82342642-180c3080-99af-11ea-8857-71e40837d41b.png">

Reply:
ya, ill move the comment!

Reply:
do you have any suggestions for the future fix? maybe returning on the api response the applied filters list or something?  we need counts to not respect status filters to enable status filtering in ui, and this is a pattern we have on many apis, so if we do find it awkward i would want to fix it everywhere